src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxnidqinterface.py

```diff
@@ -56,12 +52,18 @@ def __init__(
             Path to .nidq.bin file.
         verbose : bool, default: True
             Whether to output verbose text.
-        load_sync_channel : bool, default: False
-            Whether to load the last channel in the stream, which is typically used for synchronization.
-            If True, then the probe is not loaded.
         es_key : str, default: "ElectricalSeriesNIDQ"
         """
 
+        if load_sync_channel:
+            import warnings
+
+            warnings.warn(
+                "The load_sync_channel parameter is deprecated and will be removed in June 2025.",
+                DeprecationWarning,
+                stacklevel=2,
+            )
```

Comment:
What is the behavior change? What should be done instead?

Reply:
The last channel of the ap and lf bands of SpikeGLX binaries are used to store synch data. This command does not make sense with the NIDQ interface because there is no such channel. To be honest, I don't know what role was playing here before.

Reply:
Improved the warning to give more context.